stake-pool/py/stake_pool/instructions.py

```diff
@@ -917,3 +971,63 @@ def decrease_validator_stake(params: DecreaseValidatorStakeParams) -> Transactio
             )
         )
     )
+
+
+def create_token_metadata(params: CreateTokenMetadataParams) -> TransactionInstruction:
+    """Creates an instruction to create metadata using the mpl token metadata program for the pool token."""
+    (withdraw_authority, _seed) = find_withdraw_authority_program_address(params.program_id, params.stake_pool)
+    (token_metadata, _seed) = find_metadata_account(params.pool_mint)
+
+    keys = [
+        AccountMeta(pubkey=params.stake_pool, is_signer=False, is_writable=False),
+        AccountMeta(pubkey=params.manager, is_signer=True, is_writable=False),
+        AccountMeta(pubkey=withdraw_authority, is_signer=False, is_writable=False),
+        AccountMeta(pubkey=params.pool_mint, is_signer=False, is_writable=False),
+        AccountMeta(pubkey=params.payer, is_signer=True, is_writable=True),
+        AccountMeta(pubkey=token_metadata, is_signer=False, is_writable=True),
+        AccountMeta(pubkey=METADATA_PROGRAM_ID, is_signer=False, is_writable=False),
+        AccountMeta(pubkey=SYS_PROGRAM_ID, is_signer=False, is_writable=False),
+        AccountMeta(pubkey=SYSVAR_RENT_PUBKEY, is_signer=False, is_writable=False),
+    ]
```

Comment:
nit: I know it's not the easiest usage pattern, but to make this consistent with the other instruction creators, can you provide the ability to specify all accounts?